pytorch_lightning/callbacks/quantization.py

```diff
@@ -126,9 +133,22 @@ def custom_trigger_last(trainer):
         quantize_on_fit_end: perform the quantization in `on_fit_end`.
             Note that once converted, the model cannot be put in training mode again.
 
+        disable_observers: disable fake-quantization modules' observers during the provided stages:
+
+            - ``'validate'``. The observers will be disabled during validating.
+              Note that we don't disable observers during the sanity check as the model hasn't been calibrated with
+              training data yet. After the sanity check, the fake-quantization modules are restored to initial states.
+            - ``'test'``. The observers will be disabled during testing.
+            - ``'predict'``. The observers will be disabled during predicting.
+
+            Note that we only handle observers belonging to fake-quantization modules. When ``qconfig`` is a ``str`` and
+            ``observer_type`` is ``'histogram'``, the observers won't belong to any fake-quantization modules and will
+            not be disabled.
+
     """
 
     OBSERVER_TYPES = ("histogram", "average")
+    DISABLE_OBSERVER_STAGES = ("validate", "test", "predict")
```

Comment:
```suggestion
    ALL_DISABLE_OBSERVER_STAGES = ("validate", "test", "predict")
```
lets add also train as user may have strange use case with observing only validation

Reply:
"train" has been added. In the test `test_disable_observers`, the model is calibrated with a training batch before training if `'train'` is in `disable_observers`.